### ISSUE TITLE
"Official" download site for Mypal, with no HTTPS (or SSL3 support)

### ISSUE THREAD

**Issue by RAD750** [Feodor2/Mypal#145]
Hi there, 
I am aware there is already another site hosting Mypal that is compatible with Internet Explorer 6 (http://myapl.eu3.biz/) but it is very slow to download, you need 7-Zip (which you can't download without SSL...), Google doesn't find it and the URL is not that easy to remember by heart.

I already own a website that with some effort is now "compatible" with even very old browsers (IE 3.1, [IBM Web Browser](https://www.a-centauri.com/data/media/images/album/archeologia-dei-browser/OS2-IBM.png) from OS/2 Warp...), so I was wondering if author was OK with me: 
- registering a domain name that's easy to remember like mypal-browser.net or similar (author can of course choose the domain, I'll pay for it gladly!) or configure an existing domain
- configuring a website that supports down to SSL3 so even XP can use HTTPS, and is easily indexable by Google and other search engines
- running said website and hosting both new and old versions of Mypal for as long as it's needed. 

Storage and bandwidth are not an issue for me. Of course, I'd pay all the expenses of the "operation", and I'd like to do this just as a "thank you" for providing this awesome browser that I use daily on my old 2005 Laptop, which would be useless without Mypal :)

Thanks in advance, 
Jacopo

**Comment by ByRodrigoSoares** [comment]
The time has come for Mypal to leave Github and create his own website, server and forum.

**Comment by RAD750** [comment]
If Feodor is ok with that I'd be more than happy to host all these services and make them fully compatible with Internet Explorer

**Comment by Anbima** [comment]
But it would be important that this page remains without advertising. It would not be right for Feodor to work with MyPal and then earn another money with it.

**Comment by RAD750** [comment]
I would never put advertisement or tracking on my websites, given I always use uBlock. It would be a very hypocrite move.
You can check my website (in my github profile) if you want ;)

**Comment by Feodor2** [comment]
This very good thank you very mach, i gladly accept the proposal

**Comment by RAD750** [comment]
Thank you for accepting my offer! Do you want to choose a domain name for me to buy?

**Comment by Feodor2** [comment]
Are simple mypal names available, like mypal.org or mypal.site.
Anyway i think org or site is good,

**Comment by RAD750** [comment]
Unfortunately mypal.org and mypal.site are already taken, so I have settled for mypal-browser.org. :smiley: 
I will start work on the site immediately and notify here when it's ready for prime time!

**Comment by RAD750** [comment]
So, status update!
After much cursing I managed to produce a page which doesn't look half bad in IE6!
Please do submit feedback, especially @Feodor2 if you have specific requests like a favourite color or anything ;)

The rest of the website will follow soon.

![mypal-browser org](https://user-images.githubusercontent.com/12469744/76679431-b27f7a80-65e0-11ea-9bd9-4b7900584c2d.PNG)

**Comment by ByRodrigoSoares** [comment]
It is getting wonderful we will have a forum on the site to discuss the browser or continue on github

**Comment by RAD750** [comment]
Thank you, I'll wait for further feedback from Feodor before implementing a forum. I think it would be best to keep the issues and discussion here on Github.

The important thing, in my opinion, is that people right now that have just reinstalled XP or still using an outdated browser have no way to find and especially download the browser, because Github require TLSv1.2 and IE only supports TLS1.0 without SNI and PFS.

After the first goal is fulfilled (people can easily find Mypal and download it) I think that if they have any issue with the browser they can just use Mypal to go to Github and discuss. That way that part remains under the control of Feodor and his workflow is not disrupted.

Please tell me if my reasoning is wrong ;)

**Comment by Feodor2** [comment]
Good, but how to look the actual site, that address does not work. Also i want another text, i shall send it.
Forum involves much and frequent labor, so someone is to do it and have knowledge how. **ByRodrigoSoares** are you ready to do this?

**Comment by RAD750** [comment]
So, the website was just a test on my PC, it is now visible here: https://mypal-browser.org/demo-15-03/ (for modern browser) 
and https://mypal-browser.org/demo-15-03/index-ie6.html (for Internet Explorer 6-7-8) 
In the future, it will redirect automatically between the IE6-7-8 version and the normal version depending on what browser you use like my website www.a-centauri.com.
Of course nothing is working yet, I'm just working on the aesthetic (and please @Feodor2 feel free to send me (by email or otherwise) anything you want to put in the website, the text is something I made up just to fill the content :)) 

The forum is something I feel would be too excessive because Mypal is still a smallish project, so the amount of work necessary to moderate it, as Feodor is rightly suggesting, would be (I think) too much for just me to handle. 
For my part it isn't a problem to set it up; however I'd like to hear more opinions before actually doing it (and of course first finish the website!)

**Comment by Anbima** [comment]
Hello RAD750,
if there is no forum, then no user data is recorded. In this case, the website can be created unencrypted as http (without SSL). Then it will run safely with all old browsers. And if you make the website as pure HTML without CSS, that's optimal. Little effort and the best result.

**Comment by RAD750** [comment]
Yes, that will be my idea. I want however to keep TLS for everything except old versions of IE, because otherwise search engines will penalize the ranking. And I'd prefer for the site to look beautiful as well :)

I haven't heard back from @Feodor2 in a while, I am still waiting on the text he was supposed to send me ;)

**Comment by Feodor2** [comment]
I did not make the text yet, got a sickness. Now is coming better i returned for work.

**Comment by RAD750** [comment]
Oh wow, I hope for your recovery. Best wishes ;)

**Comment by RAD750** [comment]
Hi, any news on this? I'm still available :P

**Comment by Feodor2** [comment]
Hi
I want 2200 size text and i must confess it is hard to write good english text, i am looking someone who will write it. The thing is about old hardware that run fine but you get limited for internet usage more because other developers refuse keep winxp support, while Mypal developers not refuse...

**Comment by RAD750** [comment]
If you prefer, you can write the text in the language you prefer and I can translate it. You could even write it in a "broken" English and it is not at all an issue to clean it up and polish it for you. 
Your program deserves more attention, and I'm willing to help ;)

**Comment by segen-tra** [comment]
I can set up a forum that will work on any browser if that would help out too

**Comment by RAD750** [comment]
Site is mostly operational, still waiting for feedback or text from Feodor.
https://mypal-browser.org/

I've tested it on my XP computer and the redirections work fine. For now it is a static website but I plan to make it dynamic, so it easier to update it once I finish my university exams.

**Comment by Feodor2** [comment]
Site is pretty well, though i did not make the text. 
As for forum who will watch it as admin and moderator, i myself wont have another time for this.

**Comment by RAD750** [comment]
Thank you @Feodor2 , as soon as you want to change the text just let me know.
I am also skeptical about the usefulness of a forum, we already have the Github issues they work fine?

**Comment by Feodor2** [comment]
Text now go minor thing.
First the site has no redirecting from www, if type www.mypal-browser.org you get security error.
Next original palemoon developers declared that we abuse their sites with mypal, they even tried blocked them but failed for now #228, sure they not going to stop, so i suggest check all settings and security on the mypal site, and look on possibility to place other browser stuff like extensions themes...  may be synch too, it takes some more storage then. I can prepare all stuff myself you only have to copy it.  And that not require frequent updates.

**Comment by RAD750** [comment]
@Feodor2 The redirect, was fixed.

I still have over 10GB of free space on that machine, and if you need more I can upload the extensions, themes and everything on a separate machine that has 50GB of free space, so it is not a problem for me.

**Comment by Feodor2** [comment]
Great, i begin making addons site.

**Comment by RAD750** [comment]
Thankyou, will wait ;)

**Comment by RAD750** [comment]
@Feodor2 Any updates on the addon situation? Do you need help?
Thanks!

**Comment by Feodor2** [comment]
Extensions and dictionaries part will be ready soon.
Meanwhile it is still two copies of the site, can you set redirect? No matter with or without www but it is to leave one copy only.

**Comment by AroKol78** [comment]
Different layout same site
https://www.mypal-browser.org/ vs. http://www.mypal-browser.org/

**Comment by Feodor2** [comment]
here should leave one only too and redirect another.

**Comment by Feodor2** [comment]
I have complete extension page, http://myapl.eu3.biz/exts/, nothing excessive it has one tiny script for search only.
I can modify www.mypal-browser.org pages to add it then you upload them to site, what do you think?

**Comment by AroKol78** [comment]
**Feodor2** thanks for the
```
http://myapl.eu3.biz/exts/
http://myapl.eu3.biz/dicts/
```
(if possible, you can also add languages)

**Comment by Feodor2** [comment]
The site was abandoned.

**Comment by RAD750** [comment]
@Feodor2 Hello, sorry for disappearing but I had university exams right until now, and the pandemic didn't help at all (it was much harder for me to keep focused while not physically attending lectures)

If you can provide a file list of extensions I'll gladly add them, today I will try fixing the redirect (I am not sure why it does that and I can't replicate it on my machines) and also update to the latest version. I will also if you agree, leave you with files access so you can push releases without needing to contact me first (even though I usually handle them)

**Comment by RAD750** [comment]
Updated to latest version including langpacks

**Comment by Feodor2** [comment]
This is great, but i did not noticed your return. How do you do?
May be to close the post was bad idea, i reopen it.

**Comment by RAD750** [comment]
Download has been fixed @Feodor2 
Updated to latest version

**Comment by Feodor2** [comment]
Good, sure you may give me access.

Also shall set browsers links to www.mypal-browser.org

**Comment by RAD750** [comment]
@Feodor2 Today I will migrate site to a new server (no disruption is planned), and I need a way to contact you privately, so we can arrange delivery of the necessary certificates for you to connect and upload files.
